_posts/2017-12-21-14-01-netcore-beta1.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title : "14.1 7.x Beta1"
```

Comment:
Do we have Beta released? 
If not, the first one is better to name as "Beta".

Beta1 sounds "Yes, We will have Beta2, please wait".

Reply:
We will have a Beta2 to add batching.